logicrunner/builtin/contract/member/member.go

```diff
@@ -437,25 +426,33 @@ func (m *Member) migration(txHash string, burnAddress string, amount big.Int, un
 		for _, ref := range migrationDaemonMembers {
 			migrationDaemonConfirms[ref] = false
 		}
-		dHolder := deposit.New(migrationDaemonConfirms, txHash, amount.String(), unHoldDate)
+		migrationDaemonConfirms[m.GetReference()] = true
```

Comment:
migrationDaemonConfirms should be [3]insolar.Reference. And you should check the index of migration daemon in ActiveMD list in rd and then set ref at this index in migrationDaemonConfirms.

Reply:
this does not apply to this task.We have specific task show this -> https://insolar.atlassian.net/browse/WLT-734